docs/docs-content/clusters/edge/edgeforge-workflow/prepare-user-data.md

```diff
@@ -1,40 +1,135 @@
+| **When It's Applied** | During Edge artifact generation (before deployment).                                        | During actual Edge host installation (after ISO boot).                                              |
+| **Format**            | Key-value pairs.                                                                            | YAML in [cloud-init](https://cloudinit.readthedocs.io/) format                                      |
+| **Main Contents**     | Image architecture, OS/K8s versions, image registry information, secure boot settings, etc. | Site information, proxy settings, network interfaces, cloud-init stages, tags, registration tokens. |
+| **Focus**             | Artifact generation and packaging.                                                          | System initialization and registration.                                                             |
+| **Controls**          | What gets built and how. For example, FIPS, SELinux, installer ISO name.)                   | How the Edge host behaves post-install. For example, network config, certs, power off behavior.     |
+
+Since the `.arg` file controls the build process, you must ensure you provide the correct settings during build. If you
+provide the wrong arguments or miss any arguments, you will need to restart the build process to make changes. With
+`user-data`, you have an additional opportunity to provide additional configuration during site deployment as
+site-specific configuration. This can replace, supplement, or override your installer configuration you provide to the
+installer ISO. For more information, refer to
 [Apply Site User Data](../site-deployment/site-installation/site-user-data.md).
 
 After installation, you can also use Local UI to make edits to the user data file before you create a cluster. Refer to
 [Edit User Data](../local-ui/host-management/edit-user-data.md) for more information.
 
-This article guides you through several important configuration blocks in the **user-data** file. However, you can use
-many additional parameters to further customize your installation. Review the Edge
-[Install Configuration](../edge-configuration/installer-reference.md) resource to learn more about all the supported
+This article guides you through several important parameters in the `.arg` file and configuration blocks in the
+`user-data` file. However, you can use many additional parameters to further customize your installation. Review the
+[Edge Artifact Build Configurations](../edgeforge-workflow/palette-canvos/arg.md) and
+[Installer Reference](../edge-configuration/installer-reference.md) resource to learn more about all the supported
 configuration parameters you can use in the configuration user data.
 
-:::tip
+The [Appliance Studio](./appliance-studio.md) is a lightweight Graphic User Interface (GUI) that allows you to design
```

Comment:
I feel this paragraph needs a header or something to highlight the two approaches the user has available to them.